pulp_container/app/redirects.py

```diff
@@ -38,6 +38,13 @@ def issue_tag_redirect(self, tag):
         """
         Issue a redirect for the passed tag.
         """
+        manifest_media_type = tag.tagged_manifest.media_type
+        if (
+            manifest_media_type not in get_accepted_media_types(self.request.headers)
+            and manifest_media_type != MEDIA_TYPE.MANIFEST_V1
+        ):
+            raise Http404("Unsupported manifest version: %s" % manifest_media_type)
```

Comment:
Have you tried raising `ManifestNotFound` instead?

Reply:
hum....not yet after this change.
I'll try it!